CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 
 ## \[Unreleased\]
 
+### Removed
+
+- Remove HPO
+  (<https://github.com/openvinotoolkit/training_extensions/pull/4196>)
+- Remove Action Classification
+  (<https://github.com/openvinotoolkit/training_extensions/pull/4202>)
+
+## \[2.2.3\]
```

Comment:
There is no 2.2.3 release. Let's merge https://github.com/openvinotoolkit/training_extensions/pull/4205 first to sort out change log layout